docs/docs/.vuepress/components/GuideLink.vue

```diff
@@ -0,0 +1,121 @@
+<style lang="stylus">
+.guide-container.nodejs {
+  border-top: solid #4DA925 1em;
+}
+
+.guide-container.python {
+  border-top: solid #F5BC00 1em;
+}
+
+.guide-container.bash {
+  border-top: solid #000 1em;
+}
+
+.guide-container.go {
+  border-top: solid #00ACD7 1em;
+  text-decoration: none;
+}
+
+.guide-container:hover {
+  text-decoration: none !important;
+}
+
+.guide-container.go h3 {
+  color: #00ACD7;
+}
+
+.guide-container.bash h3 {
+  color: #000;
+}
+
+.guide-container.python h3 {
+  color: #F5BC00;
+}
+
+.guide-container.nodejs {
+  color: #4DA925;
+}
+</style>
+
+<template>
+  <div>
+    <a
```

Comment:
Not necessary at all now, but it may be worth setting up Pug so we can use Pug for templates, and so we're consistent with the basic setup on the frontend.